database/models/article.js

```diff
@@ -27,6 +27,7 @@ module.exports = (sequelize, DataTypes) => {
       },
       ratings: {
         type: DataTypes.INTEGER,
+        defaultValue: 0
```

Comment:
I don't know how you all manage to get away with no trailing commas shaa

Reply:
I try to add it as much as I can, but ESLint flags where it's required, I don't get why it's selective about the flagging sometimes